tools/fs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "fs"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+rust-version.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+tokio = { workspace = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+gloo = "0.7"
```

Comment:
We can move this to the workspace Cargo.toml too (can be done in a separate PR though if you like)